Unity-Technologies/Graphics
# Light Anchor: default behavior and remove error messages

### Purpose of this PR
- Default Distance on the LightAnchor is now 0
- Now when a scene don't have a MainCamera now we don't proce error on the console, the HelpBox in the inspector on the component.

https://fogbugz.unity3d.com/f/cases/1345263/
https://fogbugz.unity3d.com/f/cases/1345509/

### Testing status
- Add LightAnchor (check de default value).
- Change the tag of the camera to MainCamera to something else, show the inspector of the object with LightAnchor component notice we don't have the error flooding anymore.
